exchanges/gemini/gemini.go

```diff
@@ -43,8 +45,9 @@ const (
 	geminiVolume             = "notionalvolume"
 
 	// gemini limit rates
-	geminiAuthRate   = 600
-	geminiUnauthRate = 120
+	geminiRateInterval = time.Minute
+	geminiAuthRate     = 600
```

Comment:
not sure how bursting actually works n if this rate will cause problems but might wanna reduce it a little bit:
https://docs.gemini.com/rest-api/#rate-limits